app/src/main.cpp

```diff
@@ -495,6 +436,75 @@ namespace scitt
         .set_forwarding_required(ccf::endpoints::ForwardingRequired::Never)
         .install();
 
+      static constexpr auto get_entry_statement_path =
+        "/entries/{txid}/statement";
```

Comment:
Don't we want to delete the endpoints returning CBOR receipts (`/entries/{txId}/receipt`) now that we have this new endpoint for returning transparent statements?

Reply:
It's very inexpensive to expose both statements and receipts, and the latter may be useful if a caller wants to embed multiple receipts in a statement for example, so I'd vote for keeping them both.